### ISSUE TITLE
will there be support flatpak or appimage

### ISSUE THREAD

**Issue by candrapersada** [mhogomchungu/media-downloader#148]
will there be support for flathub or appimage portable on Linux? #81

**Comment by mhogomchungu** [comment]
You should have posted there.

I use PCLinuxOS and it does not support these new technologies and this makes it difficult to support them. I have been using this distribution since around 1997 and i have no plans of moving away from it.

I installed arch linux in virtualbox a while ago to investigate these technologies and i gave up after a while on arch linux because it seems to require weekly updates that are massive and it breaks when not updated for a short while.

These technologies also requires massive downloads for their runtimes and this is a bit of a problem because i pay a lot for the data that i use and there are limits to how much of my personal funds i am willing to put into these projects.

In summary, i have no objections to support them and i lack time, knowledge on how to build them, motivation and lots of data to build an environment i can use to build them.

I will support one of them or both at some point but i make no guarantees.

**Comment by shuvashish76** [comment]
Please do checkout this guide https://github.com/ivan-hc/AppImage-tips

**Comment by mhogomchungu** [comment]
Flatpak is now supported

https://flathub.org/en/apps/io.github.mhogomchungu.media-downloader

**Comment by shuvashish76** [comment]
@mhogomchungu if possible add the AppImage release repo link in the readme page.
Something like "_Unofficial_ [AppImages](https://github.com/pkgforge-dev/Media-Downloader-AppImage/releases) maintained by Package Forge devs."

**Comment by mhogomchungu** [comment]
Done
